job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/Job.groovy

```diff
@@ -160,6 +173,15 @@ abstract class Job extends Item {
     }
 
     /**
+     * Locks resources while a job is running
+     *
+     */
+    void lockableResources(@DslContext(LockableResourcesContext) Closure lockClosure = null)
```

Comment:
add a `@since` GroovyDoc tag and a `@RequiresPlugin` annotation specifying a minimum version.